src/main/java/edu/iu/terracotta/model/app/AnswerFileSubmission.java

```diff
@@ -0,0 +1,39 @@
+package edu.iu.terracotta.model.app;
+
+import edu.iu.terracotta.model.BaseEntity;
+import javax.persistence.*;
+
+@Table(name = "terr_answer_file_submission")
+@Entity
+public class AnswerFileSubmission extends BaseEntity {
+
+    @Column(name = "answer_file_submission_id", nullable = false)
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long answerFileSubmissionId;
+
+    @JoinColumn(name = "quest_sub_quest_sub_id", nullable = false)
+    @ManyToOne(optional = false)
+    private QuestionSubmission questionSubmission;
+
+
+    @Column(name = "file_content")
+    @Lob
+    private byte[] file;
```

Comment:
I think we should also store the mime type of the file. That will be helpful when redisplaying or downloading the file, which the instructor will do when grading.

Also, it would be good to store the filename which I think should be in the request as well. But that's more of a nice to have if it is easy enough.